flexmeasures/api/dev/sensors.py

```diff
@@ -63,10 +64,12 @@ def get_chart(self, id: int, sensor: Sensor, **kwargs):
         - "beliefs_after" (see the `timely-beliefs documentation <https://github.com/SeitaBV/timely-beliefs/blob/main/timely_beliefs/docs/timing.md/#events-and-sensors>`_)
         - "beliefs_before" (see the `timely-beliefs documentation <https://github.com/SeitaBV/timely-beliefs/blob/main/timely_beliefs/docs/timing.md/#events-and-sensors>`_)
         - "include_data" (if true, chart specs include the data; if false, use the `GET /api/dev/sensor/(id)/chart_data/ <../api/dev.html#get--api-dev-sensor-(id)-chart_data->`_ endpoint to fetch data)
+        - "chart_type" (currently 'bar_chart' and 'matrix_chart' are supported types)
```

Comment:
Note that this entire endpoint remains in our dev blueprint and we are already warning users that things can change here without a formal deprecation. I intend to change the names of chart types at some point in the future. Right now they map one-to-one to function names, but in the future I imagine these functions might become classes, and perhaps we define aliases. This all still needs a design session, and is not in scope right now.